webapp/src/components/Metrics/PerformanceAnalysis.tsx

```diff
@@ -130,6 +129,14 @@ const PerformanceAnalysis: React.FC<Props> = ({ jobId, pipeline }) => {
     );
   };
 
+  const VisualBarPercentage = (value: number, bgColor: string) => (
+    <VisualBar
+      formattedValue={formatRatioAsPercentageString(value as number, 1)}
```

Comment:
No need for "dangerous" `as number`
```suggestion
      formattedValue={formatRatioAsPercentageString(value, 1)}
```